libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -71,6 +71,11 @@ const AP_Param::GroupInfo AP_Scripting::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("HEAP_SIZE", 3, AP_Scripting, _script_heap_size, SCRIPTING_HEAP_SIZE),
 
+    // @Param: DEBUG_LVL
+    // @DisplayName: Enable Scripting Debug
+    // @Description: Enable scripting debug.
+    // @User: Advanced
+    // @RebootRequired: False
```

Comment:
not that it matters but we don't normally add the RebootRequired if it's false.

Reply:
We could add Values of 1:Default, 2:Enabled but this is a bit of a guess.  From a look at the code the only check is for ">1".

Reply:
It's a threshold thing, it's supposed to get more verbose as it gets higher, but apparently I've removed all the stuff that used to only kick in at 1... I might actually bump the prompt of "No scripts to run" down to the level 1 area.